src/components/report-card/ReportCard.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import fecha from 'fecha';
+
+import Card from '../ui/Card';
+
+class ReportCard extends React.Component {
```

Comment:
Do we need this component if Card component exists? Cannot remember if we discussed the need for greater complexity on card usage.